src/pay/batch_payer.py

```diff
@@ -432,6 +434,8 @@ def attempt_single_batch(self, payment_records, op_counter, dry_run=None):
                 payment_item.paid = PaymentStatus.DONE
                 logger.info("Payment to {:s} became 0 after deducting fees. Skipping.".format(payment_item.paymentaddress))
                 continue
+            else:
+                logger.info("Payment to {:s} became {:10.6f} after deducting fees.".format(payment_item.paymentaddress, pymnt_amnt / MUTEZ))
```

Comment:
I think it is better to avoid this, this will be printed for every single transaction which is a lot, furthermore if no fees are deducted this might be misleading for the user ("after deducting fees").

Reply:
Can this be just a debug log?

Reply:
I changed it to debug and added the safety check for fee amount. It would be great if you would double check/test it!

Reply:
Thanks! I'll do it ASAP.